lib/locales/el.ts

```diff
@@ -0,0 +1,493 @@
+    john_of_capistrano_priest: 'Αγίου Ιωάννη de Capestrano, πρεσβυτέρου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Οκτώβριος, p40
+    john_of_god_duarte_cidade_religious: 'Αγίου Ιωάννη του Θεού, μοναχού',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάρτιος, p33
+    john_of_kanty_priest: 'Αγίου Ιωάννη de Kety, πρεσβυτέρου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Δεκέμβριος, p42
+    john_of_the_cross_priest: 'Αγίου Ιωάννη του Σταυρού, πρεσβυτέρου και διδασκάλου της Εκκλησίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Δεκέμβριος, p42
+    john_paul_ii_pope: 'Αγίου Ιωάννη Παύλου Βʹ, Πάπα',  // src: https://episkopisyrou.gr/22-oktovriou-mnimi-tou-agiou-ioanni-pavlou-v-papa-romis/
+    john_xxiii_pope: 'Αγίου Ιωάννη ΚΓ΄, πάπα',
+    josaphat_kuntsevych_bishop: 'Αγίου Ιωσαφάτ, επισκόπου και μάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Νοέμβριος, p41
+    joseph_of_calasanz_priest: 'Αγίου Ιωσήφ de Calasanz, πρεσβυτέρου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Αύγουστος, p38
+    joseph_spouse_of_mary: 'Αγίου Ιωσήφ, Μνήστορος της Αειπαρθένου Μαρίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάρτιος, p33
+    joseph_the_worker: 'Αγίου Ιωσήφ, εργαζομένου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάϊος, p35
+    josephine_bakhita_virgin: 'Αγίας Ιωσηφίνας Bakhita, παρθένου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Φεβρουάριος, p32
+    juan_diego_cuauhtlatoatzin: 'Αγίου Ιωάννη Diego Διδάχου Cuauhtlatoatzin',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Δεκέμβριος, p42
+    justin_martyr: 'Αγίου Ιουστίνου, μάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούνιος, p36
+    lawrence_justinian: 'Αγίου Λαυρεντίου Giustiniani',
+    lawrence_of_brindisi_priest: 'Αγίου Λαυρεντίου του Βρινδησίου, πρεσβυτέρου και διδασκάλου της Εκκλησίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούλιος, p37
+    lawrence_of_rome_deacon: 'Αγίου Λαυρεντίου, διακόνου και μάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Αύγουστος, p38
+    lawrence_ruiz_and_companions_martyrs: 'Αγίου Λαυρεντίου Ruiz και συντρόφων μαρτύρων',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Σεπτέμβριος, p39
+    leo_i_the_great_pope: 'Αγίου Λέωντα του Μεγάλου, πάπα και διδασκάλου της Εκκλησίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Νοέμβριος, p41
+    louis_grignion_de_montfort_priest: 'Αγίου Λουδοβίκου Μαρία Grignion de Montfort, πρεσβυτέρου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Απρίλιος, p34
+    louis_ix_of_france: 'Αγίου Λουδοβίκου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Αύγουστος, p38
+    louise_de_marillac_religious: 'Αγίας Λουίζας de Marillac',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάρτιος, p33
```

Comment:
According to the [_Liturgical Calendar of the Archdiocese of Athens_](https://drive.google.com/file/d/1clZroPm6Gl3zeXpjk-e64LJ9fx1tm2yV/view?usp=drive_link) on p44, this celebration is now spelled as `Αγίας Λουίζας ντε Μαριγιάκ`. I think we should use this (newer) spelling, as it looks like they have updated the spelling and words to the (more) Modern Greek (at least they about this in the Preamble of this above-referenced source).

Reply:
I just want to clarify that the the preamble, indeed, mentions usage of Modern Greek but only in the refence to updated texts of Responsorial Psalms.

Reply:
> only in the refe[re]nce to updated texts of Responsorial Psalms.

What does it mean? Do the Greek still use the older celebration names everywhere else? That seems strange to me. :thinking:

I was under impression that they use the newer spelling everywhere, even hough the books printed before the modernisation of the celebration/saint names use the old spelling.